### ISSUE TITLE
scipy needs to be imported before tensorflow

### ISSUE THREAD

**Issue by niskander** [tensorflow/tensorflow#1992]
### Environment info
Operating System: 
Ubuntu 14.04.4 LTS

Installed version of CUDA and cuDNN: 
cuda-7.5
cuDNN 5

### Steps to reproduce
1. import tensorflow
2. import scipy.misc
3. scipy.misc.imread fails with "IOError: broken data stream when reading image file"

Importing scipy before tensorflow solves the issue.

**Comment by yaroslavvb** [comment]
Which TensorFlow version/Scipy version? Is there more debugging information? (ie, which line of scipy fails). I tried on last month's Anaconda install of scipy/tensorflow and it seems to work for me

**Comment by mrry** [comment]
@keveman: I'm almost certain this has the same root cause as #1924.

**Comment by niskander** [comment]
@yaroslavvb 

Since PIL is probably the culprit:
```
>>> PIL.VERSION
'1.1.7'
>>> PIL.PILLOW_VERSION
'2.3.0'
```
For tensorflow, I cloned and installed from source today
`commit 36d89cf8f7bdfb17daf93f8439d95018b8cda431`
```
image_data = misc.imread(image_path, flatten=1)
  File "/usr/lib/python2.7/dist-packages/scipy/misc/pilutil.py", line 127, in imread
    return fromimage(im,flatten=flatten)
  File "/usr/lib/python2.7/dist-packages/scipy/misc/pilutil.py", line 189, in fromimage
    im = im.convert('F')
  File "/usr/lib/python2.7/dist-packages/PIL/Image.py", line 713, in convert
    self.load()
  File "/usr/lib/python2.7/dist-packages/PIL/ImageFile.py", line 234, in load
    raise_ioerror(e)
  File "/usr/lib/python2.7/dist-packages/PIL/ImageFile.py", line 56, in raise_ioerror
    raise IOError(message + " when reading image file")
IOError: broken data stream when reading image file
```
@mrry Yeah

**Comment by eriophora** [comment]
PIL is indeed the culprit; I'm getting the same error, and I'm sidestepping scipy entirely. Even if the image is read into memory via StringIO. grr.

**Comment by mrry** [comment]
@keveman: Assigning this to you, but feel free to combine with the related bugs.

**Comment by keveman** [comment]
Fixed with 5405394
